### DISCUSSION TITLE
Fritz-NG Origin, based OS and how it works

### DISCUSSION THREAD

**Discussion by queengab** [Freetz-NG/freetz-ng#1058] [Allgemein]
Hi,

i'm Gabriele. I'm arrived here from other threads on the web about some modding for fritzbox.
In particular i was interested to give a new life to my fritzbox 3490, especially to update some features like the vpn.

So i arrived here!
I read something to understand how Fritz-NG was originated, so first of all bravo! And thanks so much!
Now i was wondering to understand better.
So freetz-NG is a for of Freetz a modded version of FritzOS and all the entire linux based system of the fritz.
Freetz NG have an alterative kernel or is only the raplacemtnt of the fritz application (front-end and backend) of the fritzbox? I think the second, am i wrong?
The Freetz-Ng is a fork of which version of the web interface of Fritz? FritOS 7.1.2. So Freetz-NG doesn't have the news features of fritzobx beacuse we can find something installed with the modded version.

Now i'm stuck! I have only my 3490 and i need the vpn, but i don't want e can't use shrewsfot vpn client.
Or I'm thinking to buy a used 7590 AX and install Fritz-NG there. I read something abput some bugs on 7590 but that were related to a bug a in the fritzos not some bugged hardware, do you know something about this?

Is there a list of all feature that Fritz-NG has more than fritz os?

Thanks!
Bye

**Comment by fda77** [comment]
![image](https://github.com/user-attachments/assets/cff90606-c8ef-457b-a2c3-96b70c7a49f5)
